browser/src/help/__snapshots__/HelpPage.spec.tsx.snap

```diff
@@ -9828,7 +9828,7 @@ question: 'Why is this variant linked to the wrong dbSNP rsID?'
 
 First, we do not continuously update the dbSNP rsIDs displayed in the browser, so there will be some differences between the databases and dbSNP depending on versions being used. The dbSNP version used are:
 
-- gnomAD v4.0: dbSNP version 156
+- gnomAD v4.0 and 4.1: dbSNP version 156
```

Comment:
Same comments as above.

Reply:
ah I actually don't know what the process is for updating these snapshots (Riley helped me with these earlier in the PR)